docs/contributing/dependencies/index.md

```diff
@@ -55,12 +92,165 @@ version is **13 days**, and **13%** of repositories have adopted this version. R
+
+> It is usually a bad idea to pin all your dependencies because it will introduce an unnecessarily
+> narrow range (one release!) and cause most users of your package to bloat their node_modules with
+> duplicates.
+
+Instead, the [recommended configuration](https://docs.renovatebot.com/presets-config/#configjs-lib)
+for libraries is to pin only `devDependencies` and leave the others as ranges.
+
+### Dependency grouping strategies
+
+We use grouping in Renovate to group related dependencies into a single PR.
+
+There are two different levels at which our grouped dependencies are configured:
+
+1. Our Renovate configuration uses the “monorepo presets” that Renovate
+   [defines](https://docs.renovatebot.com/noise-reduction/#package-grouping). These are groups of
+   dependencies that Renovate has grouped together through the `group:monorepos` configuration,
+   which we have enabled. If we use any of these pre-grouped dependencies, Renovate will
+   automatically group them together into a single PR.
+2. Grouping that we have put in place in our Renovate configuration. We have several different ways
+   that we configure to do this, which we’ll detail below.
+
+#### Why we group dependencies in our configuration
+
+If a group of dependencies are closely related, we may decide that they should move as a unit
+through the review and QA process. We’ll do this if we want to:
+
+- Group by **domain** to avoid errors resulting in inter-dependency version requirements
+  - If a dependency relies on other related dependencies to have been updated along with it, we need
+    to group these together to avoid errors when building. Sometimes we do this for all update
+    types, and sometimes we only do this for major updates, depending on when we feel like
+    inter-version dependencies exist.
+- Group by **domain** or by **package manager** to reduce the noise of too many PRs
+  - This is a [trade-off](https://docs.renovatebot.com/noise-reduction/#noise-reduction), and we
+    acknowledge that doing so will make identification of any issues stemming from the update
+    harder.
+  - Sometimes we only do this for minor or patch updates, as we acknowledge the trade-off with
+    identifying breaking changes and want to have separate PRs for major updates.
+
+:::info Grouping and team ownership
+
+Grouping does not equal team ownership. In other words, a PR that has dependencies owned by multiple
+teams would have multiple reviewers on it. As a matter of practice, we try to assign all grouped
+dependencies to a single team, so that only a single team is responsible for the review.
+
+:::
+
+### Dependency update type strategies
+
+:::info What are update types?
+
+“Update type” refers to the classification of a version as major, minor, or patch.
+
+:::
+
+### Combining update types
+
+For all of our dependencies, we default to:
+
+1. Combining minor and patch updates into a single PR for a given dependency, and
```

Comment:
💭 This feels like an unordered list to me.

Reply:
Addressed with cf9320b75d9d56cf5291b3cf3b554bce77dda3e5.